acceptance/cluster/localcluster.go

```diff
@@ -336,7 +335,6 @@ func (l *LocalCluster) createRoach(node *testNode, vols *Container, cmd ...strin
 
 	hostConfig := container.HostConfig{
 		PublishAllPorts: true,
-		NetworkMode:     container.NetworkMode(l.networkID),
```

Comment:
It's in createContainer (as mentioned in the commit message)

On Mon, Mar 7, 2016 at 4:31 PM, Peter Mattis <notifications@github.com>
wrote:

> In acceptance/cluster/localcluster.go
> <https://github.com/cockroachdb/cockroach/pull/4953#discussion_r55275608>:
>
> > @@ -336,7 +335,6 @@ func (l *LocalCluster) createRoach(node *testNode, vols *Container, cmd ...strin
> >
> >  	hostConfig := container.HostConfig{
> >  		PublishAllPorts: true,
> > -		NetworkMode:     container.NetworkMode(l.networkID),
>
> How is l.networkID being used after this removal?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/4953/files#r55275608>.
>